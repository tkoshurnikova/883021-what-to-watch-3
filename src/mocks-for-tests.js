export const FilmDetails = {
  title: `Whispering Mist`,
  genre: `Comedy`,
  year: 1995,
  video: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

export const films = [
  {
    "title": `The No Flowers`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `Rings of Cloud`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `The Sword's Memory`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `The Voyagers of the Words`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `Shores in the Heart`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `Missing Illusion`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `The Unwilling Man`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `Teacher in the Hunter`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  }
];
