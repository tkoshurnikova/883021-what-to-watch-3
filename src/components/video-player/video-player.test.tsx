import * as React from "react";
import * as renderer from "react-test-renderer";
import VideoPlayer from "./video-player";
import {films} from "../../mocks-for-tests";

it(`Render VideoPlayer`, () => {
  const tree = renderer
    .create(<VideoPlayer
      film={films[0]}
      isPlaying={true}
    />, {
      createNodeMock: () => {
        return {};
      }
    }).toJSON();

  expect(tree).toMatchSnapshot();
});
