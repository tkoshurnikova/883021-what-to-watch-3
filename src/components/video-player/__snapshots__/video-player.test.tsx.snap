// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayer 1`] = `
<video
  autoPlay={true}
  height="175"
  muted={true}
  poster="http://placehold.it/280x175"
  width="280"
/>
`;
